### PR TITLE
LPD-43258 getCompanyId returns 0 using FreeMarker

### PR DESCRIPTION
**Pull request by victorKammerer**
Jira ticket: [LPD-43258](https://liferay.atlassian.net/browse/LPD-43258)

[LPD-43258]: https://liferay.atlassian.net/browse/LPD-43258?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by victorKammerer** [comment]
ci:test:sf

**Comment by victorKammerer** [comment]
ci:test:relevant

**Comment by victorKammerer** [comment]
ci:test:sf

File: modules/apps/notification/notification-test/src/testIntegration/java/com/liferay/notification/internal/type/test/EmailNotificationTypeTest.java [hunk 1]
```diff
@@ -1504,7 +1524,8 @@ private String _getObjectEntryNotificationQueueEntryBody(
 				LanguageUtil.getLanguageId(LocaleUtil.US),
 				_portal.getPortalURL(
 					ServiceContextThreadLocal.getServiceContext(
```
Comment (linolaoj) [inline modules/apps/notification/notification-test/src/testIntegration/java/com/liferay/notification/internal/type/test/EmailNotificationTypeTest.java:1526]: Hi @victorKammerer , can we use the variable `serviceContext` here?

**Review by linolaoj — CHANGES_REQUESTED** [review]

File: modules/apps/notification/notification-test/src/testIntegration/java/com/liferay/notification/internal/type/test/EmailNotificationTypeTest.java (hunk 1 above)
Comment (victorKammerer) [reply modules/apps/notification/notification-test/src/testIntegration/java/com/liferay/notification/internal/type/test/EmailNotificationTypeTest.java]: Hey @linolaoj, we do!

**Comment by victorKammerer** [comment]
Relevant failures seems unrelated, @linolaoj

**Comment by victorKammerer** [comment]
ci:forward

**Comment by linolaoj** [comment]
ci:forward

**Comment by victorKammerer** [comment]
ci:forward

**Comment by victorKammerer** [comment]
ci:forward

**Comment by gabrielwas** [comment]
@victorKammerer you can send to Brian already

**Comment by victorKammerer** [comment]
Thanks for reviewing it @linolaoj and @gabrielwas!!

Just rebased and sent to Brian here: https://github.com/brianchandotcom/liferay-portal/pull/156697

**Pull request closed without merging**